server/src/lib/config.rs

```diff
@@ -240,6 +232,30 @@ impl<'a> Iterator for IterDevs<'a> {
     }
 }
 
+pub struct IterBlockDevs<'a> {
+    inner: btree_map::Iter<'a, String, BlockDevice>,
+}
+
+impl<'a> Iterator for IterBlockDevs<'a> {
+    type Item = (&'a String, &'a BlockDevice);
+
+    fn next(&mut self) -> Option<Self::Item> {
+        self.inner.next()
+    }
+}
```

Comment:
Is there an advantage to this wrapper type, vs returning the `btree_map:Iter` directly from `block_devices`?